streamer/src/test/utils/postgres/postgresQueries.js

```diff
@@ -0,0 +1,148 @@
+const { environment } = require('../../../environment')
+const DB_SCHEMA = environment.DB_SCHEMA
+const totalStakeThreshold = 1000000
+const totalRewardThreshold = 1000000
+const totalRewardPointsThreshold = 20
+
+
+async function getQuery(postgres, queryString) {
+    return await postgres
+        .query(queryString)
+        .catch((err) => {
+            console.log(err.stack)
+            throw new Error(`Error executing query`)
+        })
+        .then((res) => {
+            return res
+        })
```

Comment:
Let's use async/await + try/catch, instead of mixing promise and async/await syntax